python/paddle/fluid/contrib/slim/quantization/qat2_int8_mkldnn_pass.py

```diff
@@ -37,10 +37,12 @@ class Qat2Int8MkldnnPass(object):
 
     def __init__(self,
                  _quantized_ops,
+                 _scale_from_attr=False,
```

Comment:
Whether scales are collected from attribute by default? please consider to delete this input params.

Reply:
@juncaipeng , do you mean the priorities of sources of scales should be set without giving an option to change it?

Reply:
Done.